driver/csiplugin/controllerserver.go

```diff
@@ -941,7 +941,12 @@ func (cs *ScaleControllerServer) copySnapContent(scVol *scaleVolume, snapId scal
 		return err
 	}
 
-	targetFsDetails, err := conn.GetFilesystemDetails(targetFsName)
+	primaryConn, isprimaryConnPresent := cs.Driver.connmap["primary"]
+	if !isprimaryConnPresent {
+		glog.Errorf("unable to get connector for primary cluster")
+		return status.Error(codes.Internal, "unable to find primary cluster details in custom resource")
+	}
+	targetFsDetails, err := primaryConn.GetFilesystemDetails(targetFsName)
```

Comment:
primary cluster won't be aware of targetFsName as it is name of filesystem on remote cluster

Reply:
Removed the change.